runtime/lua/vim/loader.lua

```diff
@@ -200,7 +200,7 @@ function Loader.loader(modname)
     return chunk or error(err)
   end
   Loader._hashes = nil
-  return '\ncache_loader: module ' .. modname .. ' not found'
+  return ("\n\tcache_loader: module '%s' not found"):format(modname)
```

Comment:
could you also add a test case in `test/functional/lua/load_spec.lua` ? something like

```lua
  it('correct indent on error message', function()
    exec_lua [[
      vim.loader.enable()
    ]]
    exec_capture("lua require('foo')")
  end)
```

Reply:
Sure! Sorry, I couldn't find loader_spec file when I created the PR...

Reply:
internal assertions is wired. you can capture the output, simply case like 
```lua
 it('correct indent on error message (#29809)', function()
   local result = exec_lua [[
      vim.loader.enable()
      local _, errmsg = pcall(require, 'non_existent_module')
      errmsg = vim.split(errmsg, '\n')
      return {errmsg[3], errmsg[4]}
    ]]
    eq({'\tcache_load: module non_exit_module not found','\tcache_load_lib: module non_exit_module not found'}, result)
  end)
```